docs/assets/docs.css

```diff
@@ -87,3 +87,35 @@ span.hljs-meta {
 .example-source-container {
     margin-bottom: 2rem;
 }
+
+.layout-demo .row {
```

Comment:
I have used 4 space layout so far. I actually think 2 is more consistent with what we have, so I suggest changing everything to 2?

Reply:
Sounds good, I'll add that to this PR